WcaOnRails/app/views/static_pages/disclaimer.html.erb

```diff
@@ -3,11 +3,5 @@
 <div class="container">
   <h1><%= yield(:title) %></h1>
```

Comment:
This `h1` can also be moved into React. If you need to access the title, you can pass it to the React props via the second argument (the one that's currently an empty object.)

Reply:
Although arguably, you could also hard-code "Disclaimer" in this case, because we're going to keep this page in English anyways and not localize it for now (see also my other comment below)